### DISCUSSION TITLE
How to sync bookmarks ?

### DISCUSSION THREAD

**Discussion by Ayaaz13** [vrtmrz/obsidian-livesync#531] [General]
Hi,

I just installed this plugin and it works great.
I was able to sync everything between my PC and phone, except for bookmarks.

Tried to play with customization sync, but got no results.
So what should I do to sync bookmarks ? Or is this not possible ?

**Comment by vrtmrz** [comment]
Sorry for being late! 
I think updating `booksmarks.json` via the Customisation Sync might resolve that. Have you tried it? 

(Or, possibly rebooting Obsidian before that Obsidian wrote it after we modified, is required)...

**Reply by titandrive** [reply]
Just came across this. Would you be able to provide more info on how to do this?
